Sources/SwiftJWT/JWTSigner.swift

```diff
@@ -49,35 +66,62 @@ public struct JWTSigner {
     }
     
     /// Initialize a JWTSigner using the RSA 256 bits algorithm and the provided privateKey.
+    /// - Parameter privateKey: The UTF8 encoded PEM private key.
```

Comment:
You added the words "including the BEGIN PUBLIC KEY header" in the case of JWTVerifier - whilst I understand this is to disambiguate between key and certificate, I think it would be nice to include the statement about the header here too (and ditto for the EC keys for sign/verify)